api/apps/v1alpha1/nemo_datastore_types.go

```diff
@@ -649,10 +644,7 @@ func (n *NemoDatastore) GetDefaultStartupProbe() *corev1.Probe {
 		ProbeHandler: corev1.ProbeHandler{
 			HTTPGet: &corev1.HTTPGetAction{
 				Path: "/v1/health/ready",
-				Port: intstr.IntOrString{
-					Type:   intstr.Type(0),
-					IntVal: n.Spec.Expose.Service.Port,
-				},
+				Port: getProbePort(n.Spec.Expose.Service),
```

Comment:
`GetDefaultStartupProbe` should be called by `GetStartupProbe`
It isn't the case currently, we should make that change